package/itest/ubuntu.sh

```diff
@@ -43,18 +43,12 @@ apt-get install -y --force-yes python3.7 python3-pip python3-yaml
 # we also install python3-distutils here to avoid issues on newer ubuntus
 # where disutils isn't included with python (and even though clusterman depends on it, the right
 # version isn't installed in this itest container)
-if  [ "${DISTRIB_CODENAME}" != "xenial" ]; then
 apt-get install -y --force-yes python3.7-distutils
```

Comment:
we upgrade to python3.8 elsewhere but not here?

Reply:
right, thanks for the catch!

Reply:
updated